client/src/components/admin/AdminSites.vue

```diff
@@ -6,30 +6,47 @@
         <table class="table table-striped table-hover text-sm">
           <thead>
             <th>Name</th>
-            <th>Startrichtung</th>
-            <th>Website</th>
-            <th>Verein</th>
-            <th>Lat</th>
-            <th>Long</th>
-            <th>Hochgeladen am</th>
-            <th>Nachricht an Pilot</th>
-            <th>Akzeptieren</th>
-            <th>Gebiet löschen</th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
+            <th></th>
           </thead>
           <tbody>
             <tr v-for="site in sites" :key="site.id" :item="site">
-              <td>{{ site.name }}</td>
-              <td>{{ site.direction }}</td>
-              <td>{{ site.website }}</td>
-              <td>{{ site.club?.name }}</td>
-              <td>{{ site.point?.coordinates[0] }}</td>
-              <td>{{ site.point?.coordinates[1] }}</td>
-              <td>
+              <td data-cy="site-name">{{ site.name }}</td>
+              <td data-cy="site-direction">{{ site.direction }}</td>
+              <td data-cy="site-elevation">{{ site.elevation }}m</td>
+              <td data-cy="site-heightDifference">
+                <i class="bi bi-arrow-bar-down"></i>
```

Comment:
Ich hätte das vorgeschlagen... bi bi-graph-down-arrow ;-)

Musste gerade in den Quellcode schauen, um ganz sicher zu gehen, was sich hinter dem Symbol nochmal verbirgt ^^